org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/serializer/SerializerFragment2.xtend

```diff
@@ -534,7 +534,7 @@ import org.eclipse.xtext.util.internal.Log
 		}
 		return '''
 			/**
-			 * �NodeModelUtils.getNode(rule).textWithoutComments.trim.replace('\n', '\n* ')�
+			 * �NodeModelUtils.getNode(rule).textWithoutComments.trim.replace('\n', '\n* ').replace('*/','*&#47;')�
```

Comment:
since this is Java and not XML we might want to use `'*\u2F` rather than '*&#47;'